interfaces/ac_rcd.yaml

```diff
@@ -0,0 +1,27 @@
+description: >-
+  This interface defines an AC Residual Current Monitor (RCD). Actual emergency switch off is done in HW directly,
+  but this interface allows some control and telemetry.
```

Comment:
I think we should rather write *privides

Reply:
changed